src/itemdb.d

```diff
@@ -91,6 +81,34 @@ final class ItemDatabase
 		deleteItemByIdStmt = db.prepare("DELETE FROM item WHERE driveId = ? AND id = ?");
 	}
 
+	void createTable(int dbVersion)
+	{
+		db.exec("CREATE TABLE item (
+				driveId          TEXT NOT NULL,
+				id               TEXT NOT NULL,
+				name             TEXT NOT NULL,
+				type             TEXT NOT NULL,
+				eTag             TEXT,
+				cTag             TEXT,
+				mtime            TEXT NOT NULL,
+				parentId         TEXT,
+				crc32Hash        TEXT,
+				sha1Hash         TEXT,
+				quickXorHash     TEXT,
+				remoteDriveId    TEXT,
+				remoteId         TEXT,
+				deltaLink        TEXT,
+				PRIMARY KEY (driveId, id),
+				FOREIGN KEY (driveId, parentId)
+				REFERENCES item (driveId, id)
+				ON DELETE CASCADE
+				ON UPDATE RESTRICT
+			)");
+		db.exec("CREATE INDEX name_idx ON item (name)");
+		db.exec("CREATE INDEX remote_idx ON item (remoteDriveId, remoteId)");
+		db.setVersion(itemDatabaseVersion);
```

Comment:
See comment about the method definition above

Reply:
noted